### PR TITLE
get comments for a post

### PR DESCRIPTION
**Pull request by emir4ik-code**

### CODE REVIEW

File: src/controllers/comment.controller.ts [hunk 1]
```diff
@@ -27,6 +25,16 @@ class CommentController {
       next(error);
     }
   };
+
+  public getCommentsForPost = async (req: Request, res: Response, next: NextFunction) => {
+    try {
+      const postId = Number(req.params.post_id);
```
Comment (ZohaibManzoor00) [inline src/controllers/comment.controller.ts:31]: change postId to id instead

File: src/services/comment.service.ts [hunk 2]
```diff
@@ -17,6 +17,10 @@ class CommentService {
     const updatedComment = await updateAndReturn<BaseCommentDto, UpdateCommentRequestDto>(comment_id, commentData, CommentEntity);
     return updatedComment;
   }
+  public async getCommentsForPost(postId: number): Promise<BaseCommentDto[]> {
```
Comment (ZohaibManzoor00) [inline src/services/comment.service.ts:20]: change to id

File: src/controllers/comment.controller.ts (hunk 1 above)
Comment (emir4ik-code) [reply src/controllers/comment.controller.ts:31]: nah

**Review by Matt-Tan15 — DISMISSED** [review]

File: src/services/comment.service.ts (hunk 2 above)
Comment (emir4ik-code) [reply src/services/comment.service.ts]: changed

File: src/controllers/comment.controller.ts (hunk 1 above)
Comment (emir4ik-code) [reply src/controllers/comment.controller.ts]: changed

**Pull request closed without merging**